### DISCUSSION TITLE
how to run `helix` commands at startup?

### DISCUSSION THREAD

**Discussion by amtoine** [helix-editor/helix#4138] [Q&A] [answered]
hello again :wink: 

i would like to change the `indent-style` to using `4` whitspaces instead of `tabs`
i can achieve that effect by running `:indent-style 4` from inside `helix` :+1: 

**is there a way to run commands each time `helix` launches?**

i thought it could be in the [hooks](https://docs.helix-editor.com/hooks.html), but this section of the documentation is empty right? and does not even exist on the `master` branch documentation :open_mouth:

**Comment by the-mikedavis** [comment, marked as answer]
This is not currently possible. It's something that hooks would cover but hooks are not implemented.

**Reply by amtoine** [reply]
then we'll have to wait a bit more :wink: 

thanks for the answer :relieved: 

@the-mikedavis, it's something i would really love to have available, **is there anything i can do to help?** :yum:

**Comment by chorcheus** [comment]
Hi,
as a workaround you could use custom configuration
```
[keys.normal]
i = [":indent-style 4", "insert_mode"]
```
but this somehow creates longer and longer indents on each of the following lines - which seems to be a bug. 
Maybe fixing this could take less time than implementing hooks.

Thanks for all the work!

**Reply by chorcheus** [reply]
Alternatively, one could edit language configuration and set tab options there: https://docs.helix-editor.com/languages.html

**Reply by amtoine** [reply]
interesting workaround :open_mouth: 
thanks @chorcheus :star_struck:
